Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Registration.hs

```diff
@@ -129,6 +129,7 @@ auth req mbBundleVersion mbClientVersion = do
       countryCode = req.mobileCountryCode
   mobileNumberHash <- getDbHash mobileNumber
   let merchantId = Id req.merchantId :: Id DO.Merchant
+  whenM (isJust <$> QP.checkForAlternateNumber countryCode mobileNumberHash merchantId) $ throwError $ InvalidRequest "Access Denied"
```

Comment:
400 error can not be Access Denied, fix either description or error code

Reply:
Why to have this check? Let driver login through their alternate phone number too.

Reply:
if person other than driver tries to login through alternate number then the driver gets logged out. And that person can use his credentials of DL , Rc . Is it expected ?